tentacle/src/service.rs

```diff
@@ -724,7 +720,8 @@ where
         let handles = self.session_handles_open(self.next_session);
 
         let mut by_name = HashMap::with_capacity(self.protocol_configs.len());
-        let mut by_id = HashMap::with_capacity(self.protocol_configs.len());
+        let mut by_id =
```

Comment:
unnecessary change?

Reply:
https://doc.rust-lang.org/nightly/std/collections/struct.HashMap.html#method.with_capacity

`with_capacity` only impl for `RandomState` hasher